modules/ROOT/pages/troubleshooting-anypoint-platform-access.adoc

```diff
@@ -37,15 +37,15 @@ Please create a https://help.mulesoft.com[MuleSoft support case] for account del
 
 === What account should I use?
 
-If you have one or more existing MuleSoft community accounts tied to the email address at which you received your invitation, MuleSoft prompts you to use your existing account to sign in and accept the invitation. You can also choose to create a new account by following the Sign Up link at the bottom of the form.
+If you have one or more existing MuleSoft community accounts tied to the email address at which you received your invitation, MuleSoft prompts you to use your existing account to log in and accept the invitation. You can also choose to create a new account by following the *Sign Up* link at the bottom of the form.
 
 === How can I use a different email address?
 
 Accept the invitation using the email where it was sent. After you've accepted the invitation, visit your profile to change your email address.
 
 === Why can't I see anything?
 
-If you accept an invitation, sign in, and can't see anything on your organization's Developer Portal, one of the following may be affecting your view:
+If you accept an invitation, log in, and can't see anything on your organization's Developer Portal, one of the following may be affecting your view:
```

Comment:
Search for may and see if it really should be "might" or "can"

Reply:
```suggestion
If you accept an invitation, log in, and can't see anything on your organization's Developer Portal, one of the following might be affecting your view:
```